runtime/lua/man.lua

```diff
@@ -1,96 +1,90 @@
 local api, fn = vim.api, vim.fn
 
-local FIND_ARG = '-w'
-local localfile_arg = true -- Always use -l if possible. #6683
-
----@type table[]
-local buf_hls = {}
-
 local M = {}
 
-local function man_error(msg)
-  M.errormsg = 'man.lua: ' .. vim.inspect(msg)
-  error(M.errormsg)
-end
-
--- Run a system command and timeout after 30 seconds.
----@param cmd string[]
----@param silent boolean?
----@param env? table<string,string|number>
----@return string
+-- Run a system command and timeout after 10 seconds.
+--- @param cmd string[]
+--- @param silent boolean?
+--- @param env? table<string,string|number>
+--- @return string
 local function system(cmd, silent, env)
   local r = vim.system(cmd, { env = env, timeout = 10000 }):wait()
 
   if not silent then
     if r.code ~= 0 then
       local cmd_str = table.concat(cmd, ' ')
-      man_error(string.format("command error '%s': %s", cmd_str, r.stderr))
+      error(string.format("command error '%s': %s", cmd_str, r.stderr))
     end
     assert(r.stdout ~= '')
   end
 
   return assert(r.stdout)
 end
 
----@param line string
----@param linenr integer
-local function highlight_line(line, linenr)
-  ---@type string[]
+--- @enum Man.Attribute
+local ATTRS = {
+  NONE = 0,
+  BOLD = 1,
+  UNDERLINE = 2,
+  ITALIC = 3,
```

Comment:
even in C we no longer use uppercase for enums. can we avoid that for Lua too.

Reply:
I'd known that to get a comment. 😛 We've already discussed this at length so not sure if we should revisit. I find using this convention to mark constants helps readability. C has its own convention of `k*`, which I'm sure you don't want to replicate here?

We already do this for `vim.diagnostic.severity`, `vim.lsp.log.levels`, and other stuff. 

I can switch to 🐍 case if that's more acceptable? `_snipper_grammar.lua` and `lsp/protocol.lua` do that.